SpatialGDK/Source/SpatialGDK/Private/Utils/ComponentFactory.cpp

```diff
@@ -447,6 +469,22 @@ TArray<FWorkerComponentUpdate> ComponentFactory::CreateComponentUpdates(UObject*
 				OutBytesWritten += BytesWritten;
 			}
 		}
+
+		if (Info.SchemaComponents[SCHEMA_InitialOnly] != SpatialConstants::INVALID_COMPONENT_ID)
+		{
+			if (!Info.bDynamicSubobject || !bInitialOnlyReplicationEnabled)
```

Comment:
Could you clarify what this condition is supposed to do? It seems like it would replicate initial only data in all cases except when it's a dynamic subobject **and** initial only replication is enabled. What is the expected behavior when initial only replication is disabled?

Reply:
> except when it's a dynamic subobject and initial only replication is enabled
Yep that's the intention - I'll add a comment.